models/department.go

```diff
@@ -21,3 +21,13 @@ func DepartmentsToBases(deps []Department) []DepartmentBase {
 	}
 	return bases
 }
+
+// convert department UUIDs into Departments
```

Comment:
I'm not sure why the linter does not fail, but [this is against conventions](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences)